api/src/main/java/org/eclipse/microprofile/metrics/SimpleTimer.java

```diff
@@ -94,7 +94,7 @@
      * 
      * @return the elapsed time {@link java.time.Duration duration}
      */
-    Duration getElapsedTime();
+    Duration getSum();
```

Comment:
see my comment in the conversation on this.  Now that we've moved away from having Summable interface, there's not much reason to change Timer/SimpleTimer method name from getElaspedTime to getSum.  I'd suggest we avoid the backward-compatibility-breaking change and leave it as getElapsedTime.